.github/workflows/ci_workflow.yml

```diff
@@ -354,35 +430,47 @@ jobs:
   #       Debug build testing with Python bindings and docs enabled.
 
   windows:
-    name: 'Windows 2019 MSVC 16.4
-      <config=${{ matrix.build-type }}, 
+    name: 'Windows 2019 
+      <MSVC 16.4,
+       config=${{ matrix.build-type }}, 
        shared=${{ matrix.build-shared }}, 
        sse=${{ matrix.use-sse }}, 
+       cxx=${{ matrix.cxx-version }}, 
        python=${{ matrix.python-version }}, 
        docs=${{ matrix.build-docs }}>'
     runs-on: windows-2019
     strategy:
       matrix:
-        build: [1, 2, 3]
+        build: [1, 2, 3, 4]
         include:
           - build: 1
             build-type: Release
             build-shared: 'ON'
             build-docs: 'ON'
             use-sse: 'ON'
+            cxx-version: 11
             python-version: 3.7
           - build: 2
             build-type: Debug
             build-shared: 'ON'
             build-docs: 'OFF'
             use-sse: 'ON'
+            cxx-version: 11
             python-version: 3.7
           - build: 3
             build-type: Release
             build-shared: 'OFF'
             build-docs: 'ON'
             use-sse: 'OFF'
+            cxx-version: 11
             python-version: 2.7
+          - build: 4
+            build-type: Release
+            build-shared: 'ON'
+            build-docs: 'ON'
```

Comment:
Build 1 is already generated the docs so it could be OFF for build 3 & 4.

Reply:
Done.